### ISSUE TITLE
Question: How to generate msvc project files with progen?

### ISSUE THREAD

**Issue by JackyWoo** [pocoproject/poco#3745]
How to generate msvc project files with progen? Can it be generated on MacOS?

**Comment by aleks-f** [comment]
No, Progen requires Windows and Visual Studio 2017, 2019 or 2022. It uses VS_90 project as a template for generation of target project.
